app/scripts/pages/explore/explore-component.jsx

```diff
@@ -198,7 +207,7 @@ class ExplorePage extends PureComponent {
                   className="toggle-status"
                   onClick={() => this.onToggleSidebar()}
                 >
-                  {isSidebarHidden ?
+                  {sidebar.open ?
                     <Icon name="icon-arrow-right" className="-medium" /> :
                     <Icon name="icon-arrow-left" className="-medium" />}
```

Comment:
You can use `classnames` here for the `name` attribute instead of choosing between two different elements. Just a suggestion.